src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -123,88 +126,66 @@ Here's a screenshot of 3 terminal sessions showing k6, iftop and htop.
-> 
-> When running a ES5.1 script instead of the original ES6+ script, k6 can use about 50-85% of memory and significantly reduce the CPU load and startup time.
-
-You can use [webpack](https://webpack.js.org/) to transpile the scripts outside of k6. We have prepared a [webpack.config example](https://github.com/grafana/k6-hardware-benchmark/blob/master/webpack.config.js) that transforms ES6+ code to ES5.1 code for k6. 
-
-In the [k6-hardware-benchmark](https://github.com/grafana/k6-hardware-benchmark) repository, you can use it as follows: 
-
-```bash
-git clone https://github.com/grafana/k6-hardware-benchmark/
-cd k6-hardware-benchmark
-yarn install
-
-yarn run to-es5 someplace/yourscript.js
-# builds the ES5 script in someplace/yourscript.es5.js
-
-k6 run --compatibility-mode=base someplace/yourscript.es5.js
-```
-
 ### discardResponseBodies
 
-You can tell k6 to not process the body of the response by setting `discardResponseBodies` in the options object like this:
+k6 by default loads the response body of the request into memory. This causes much higher memory consumption and is often not necessary.
+You can tell k6 to not process the body of all responses by setting `discardResponseBodies` in the options object like this:
 
 ```javascript
 export const options = {
   discardResponseBodies: true,
 };
 ```
 
-k6 by default loads the response body of the request into memory. This causes much higher memory consumption and often is completely unnecessary.
-If you need response body for some requests you can set [Params.responseType](/javascript-api/k6-http/params).
+If you need the response body for some requests you can override this with [Params.responseType](/javascript-api/k6-http/params).
+
 
 ### --no-thresholds --no-summary
 
-If you are running a local test and streaming results to the cloud (`k6 run -o cloud`), you may want to disable the terminal summary
-and local threshold calculation because thresholds and summary will be displayed in the cloud.
+If you are running a local test and streaming results to the k6 Cloud (`k6 run -o cloud`), you may want to disable the terminal summary
+and local threshold calculation because thresholds and summary will be displayed in the Cloud.
 This will save you some memory and CPU cycles.
 
 Here are all the mentioned flags, all in one:
 
 ```bash
-k6 run scripts/website.es5.js \
+k6 run scripts/website.js \
   -o cloud \
   --vus=20000 \
   --duration=10m \
-  --compatibility-mode=base \
   --no-thresholds \
-  --no-summary \
+  --no-summary
 ```
 
-### Remove unnecessary checks, groups and custom metrics
+### Test script optimizations
 
-If everything else has failed and you are trying to squeeze more performance out of the hardware,
-you can consider optimizing the code of the load test itself.
+If you are trying to squeeze more performance out of the hardware, you can consider optimizing the code of the test script itself.
```

Comment:
I think each of these sections should mention how the proposed change would manifest in terms of system performance. If all improve memory, then we could retitle the heading: "Script optimizations to reduce memory"

Reply:
Also, some of these features are quite important. It raises the question, if users have to disable them, is it worth running the test. Why should someone run such a test?